### DISCUSSION TITLE
14. Добавьте HTTP-коммуникацию ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#72] [Announcements]
# 14. Добавьте HTTP-коммуникацию ⚡️ Angular с примерами кода

Этот урок демонстрирует, как интегрировать HTTP и API в ваше приложение

https://angdev.ru/angular/first-app-lesson-14/

**Comment by Begimai02** [comment]
Очень классное приложение для новичков 🔥. Всё получилось. Правда картинки я оставил так, как было до json servera. Иначе пришлось бы скачивать картинки, а теперь они подтягиваются из другого сервера.
